packages/perspective-jupyterlab/package.json

```diff
@@ -39,7 +39,7 @@
         "@finos/perspective-viewer-d3fc": "^2.3.2",
         "@finos/perspective-viewer-datagrid": "^2.3.2",
         "@finos/perspective-viewer-openlayers": "^2.3.2",
-        "@jupyter-widgets/base": "^2 || ^3 || ^4 || ^5 || ^6",
+        "@jupyter-widgets/base": "^6",
```

Comment:
https://ipywidgets.readthedocs.io/en/latest/migration_guides.html#updating-package-json

"Note that “@jupyter-widgets/base” version 5 is reserved for ipywidgets 7 support on JupyterLab 4, “@jupyter-widgets/base” version 6 is the version released with ipywidgets 8."

Reply:
Yeah I think we want ipywidgets 8 (and base version 6). The extension's `PerspectiveJupyterWidget` class calls `this._view.processLuminoMessage`, a method which, according to that document, is not present in ipywidgets 7, where it was called `processPhosphorMessage`. 

https://ipywidgets.readthedocs.io/en/latest/migration_guides.html#phosphor-lumino

If we do currently support ipywidgets I'd like to know how, because I haven't seen any stub code in perspective to support `processPhosphorMessage`

Reply:
We currently support ipywidgets 7

Reply:
No, we don't. We say we support ipywidgets 7 but it's a lie. 

> The extension's PerspectiveJupyterWidget class calls this._view.processLuminoMessage, a method which, according to that document, is not present in ipywidgets 7, where it was called processPhosphorMessage.

Reply:
if we do want to support ipywidgets 7 we'll need something like this. Not a big deal to add it but right now today HEAD of master won't work with ipywidgets 7.

https://github.com/jupyter-widgets/ipyleaflet/pull/969

Reply:
If someone is successfully using perspective in a notebook with ipywidgets 7 I would be very interested in learning how they've done that.

Reply:
I believe this commit broke support for ipywidgets 7 https://github.com/finos/perspective/commit/715cd7ad2a77cc6a8a7cf39e85ab618e6163e288

Reply:
https://github.com/finos/perspective/pull/2325/files#diff-6653218916957fa967fc60ecfce389d2126af0e69bac83f49a29a2a38a6ea8daR59